travis-run-tests.sh

```diff
@@ -1,8 +1,10 @@
 #! /usr/bin/env bash
 
+set -e
+
 current_version=`python setup.py --version`
 
-if ! git diff-index --quiet master openfisca_web_api
+if [[ $(git --no-pager diff --ignore-blank-lines  master openfisca_web_api) ]]
```

Comment:
I tried and it didn't work (I removed the `--quiet` option here to see the results):

```sh
echo >> openfisca_web_api/environment.py
 openfisca  ~/D/o/openfisca-web-api (master|✚1) $ git diff-index  master openfisca_web_api 
:100644 100644 14290779c09faf9c187970725cda513f69f8cce8 0000000000000000000000000000000000000000 M	openfisca_web_api/environment.py
 openfisca  ~/D/o/openfisca-web-api (master|✚1) $ git diff-index --ignore-blank-lines master openfisca_web_api 
:100644 100644 14290779c09faf9c187970725cda513f69f8cce8 0000000000000000000000000000000000000000 M	openfisca_web_api/environment.py
 openfisca  ~/D/o/openfisca-web-api (master|✚1) $ git diff-index --ignore-all-space master openfisca_web_api 
:100644 100644 14290779c09faf9c187970725cda513f69f8cce8 0000000000000000000000000000000000000000 M	openfisca_web_api/environment.py
```